tensorzero-internal/src/inference/providers/gcp_vertex_gemini.rs

```diff
@@ -961,9 +962,45 @@ struct GCPVertexGeminiResponseContent {
 }
 
 #[derive(Debug, Deserialize, Serialize)]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+enum GCPVertexGeminiFinishReason {
```

Comment:
`serde::other` to some unknown entry so we don't blow up if they introduce a new reason